### DISCUSSION TITLE
Select Multiple Fields in Dropdown

### DISCUSSION THREAD

**Discussion by jacklythgoee** [twbs/bootstrap#35478] [Q&A]
Hi all,

I was wondering if this package includes a dropdown that you can select multiple fields for example.

I am currently having a problem with a form webpage i am creating. 

If a customer has ordered 2 machines and they are both different i want to be able to select them both in the same field not making another.

Currently i can only select one field. I have tried countless amount of ideas and plugins including the chosen plugin and it just seems to not be working...

I want the following to look similar to this photo i have linked.
https://imgur.com/qsYYm2i

**Comment by gerardnico** [comment]
Nope.

Have you tried this one ?
https://github.com/snapappointments/bootstrap-select/

**Reply by jacklythgoee** [reply]
Can you guide me in the installation and setup?

**Reply by jacklythgoee** [reply]
I have just went through the setup and followed the same code that they used. I used their CDNs and it isnt working...

**Reply by gerardnico** [reply]
Example:
https://datacadamia.com/web/ui/bootstrap/select#example

**Reply by jacklythgoee** [reply]
This works! Thank you. Unsure why it wouldnt work with the tags from their template.

Where would i be able to customise the menu?

**Reply by gerardnico** [reply]
The html element for the data and the styling otherwise.
Example are [here](https://developer.snapappointments.com/bootstrap-select/examples/)

**Reply by jacklythgoee** [reply]
Hi this is great thank you!
Unsure if i am just missing it but om trying to make the "buttons" smaller so they can all fit on one vertical line. Where is this located?

**Reply by gerardnico** [reply]
Sorry I don't give professional support. You seems like a beginner. I would ask advice from a colleague.

**Reply by jacklythgoee** [reply]
i know how to work myself around a html and css page, ive made a good one that im using this in. I just have no clue how to use these script plugins and things related closely to that area.

**Reply by gerardnico** [reply]
Check the doc then. https://getbootstrap.com/docs/5.1/components/buttons/

**Reply by jacklythgoee** [reply]
Thanks,

I have also noticed that the plugins have been messing up with my written css how can i mitigate this?

**Reply by gerardnico** [reply]
https://datacadamia.com/css/cascading
